mingw-w64-git/git-wrapper.c

```diff
@@ -621,6 +627,20 @@ static void initialize_top_level_path(LPWSTR top_level_path, LPWSTR exepath,
 				return;
 			}
 			top_level_path[len] = L'\0';
+			if (running_on_arm64) {
+				WCHAR msystem_bin2[64];
+
+				/* Maybe we are on ARM64 but don't have the binaries? */
+				swprintf(msystem_bin2, sizeof(msystem_bin2), L"mingw%d\\bin", (int) sizeof(void *) * 8);
```

Comment:
Shouldn't this check for the `arm64\bin` folder instead? If that doesn't exist, we're running on ARM64 but don't have the binaries, correct?

Reply:
Yes, and we're already in the clause where we determined that `arm64/bin` does not exist. So we're falling back, looking for `mingw<bitness>/bin` here.